Source/OxyPlot/PlotModel/PlotModel.Rendering.cs

```diff
@@ -433,6 +433,13 @@ private void RenderSeries(IRenderContext rc)
         /// <param name="rc">The render context.</param>
         private void RenderTitle(IRenderContext rc)
         {
+            OxySize? maxSize = null;
+
+            if (ClipTitle)
```

Comment:
This should be

            if (this.ClipTitle)

To satisfy style guidelines.